pyke/lightcurve.py

```diff
@@ -25,15 +33,53 @@ def __init__(self, time, flux, flux_err=None, quality=None, centroid_col=None,
         self.centroid_col = centroid_col
         self.centroid_row = centroid_row
 
-    def detrend(self, method='arclength', **kwargs):
+    def stitch(self, *others):
+        time = self.time
```

Comment:
We may think about supporting multiple "stitching methods" with an optional `method=` attribute, but it's beyond the scope of this PR!